lms/djangoapps/courseware/management/commands/tests/test_dump_course.py

```diff
@@ -55,15 +55,19 @@ def load_courses(self):
         # Add a course with a unicode name, if the modulestore
         # supports adding modules.
         if hasattr(store, 'create_xmodule'):
```

Comment:
Right, so this part of the test hadn't been run since create_xmodule was renamed to create_xblock.

I'll push up a commit to change this to create_xblock, but one interesting thing is that with split modulestore, I get errors about duplicate courses (this code is run for every test point, and the name of the created course is always the same), but I don't get those errors for old mongo. So in my change, I am also making the "org" unique per test point.